benchexec/tools/fizzer.py

```diff
@@ -92,6 +100,13 @@ def determine_result(self, run):
         if not run.output:
             return result.RESULT_UNKNOWN
 
+        try:
+            mark = "--- TestCompResult ---"
+            txt = run.output.text
+            return txt[txt.index(mark) + len(mark) :].strip()
```

Comment:
This returns everything after the mark, which could be a multi-line string, for example if for some reason the tool crashes after printing the output. That would probably break the tables. Could it be improved such that only the actually desired result is extracted, for example by restricting it to the remainder of the line or the next line, depending on how the output is structured?

Reply:
Ok, fixed.